.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: go
 sudo: false
 go:
- - "1.10.1"
+ - 1.12.5
```

Comment:
I guess it could be wise to start testing backward compatibility 1.11, 1.10, etc
I'm not a huge fan of testing only 1.12.5, as it's cutting edge and even on Debian test it's at 1.11.6